docs/src/main/sphinx/release/release-465.md

```diff
@@ -0,0 +1,80 @@
+# Release 465 (18 Nov 2024)
+
+## General
+
+* Add the {func}`cosine_similarity` function for dense vectors. ({issue}`23964`)
+* Add support for reading geometries in [EWKB
+  format](https://en.wikipedia.org/wiki/Well-known_text_representation_of_geometry)
+  with the {func}`ST_GeomFromBinary` function. ({issue}`23824`)
+* Add support for parameter of `bigint` type for the {func}`repeat` function. ({issue}`22867`)
+* Add support for the `ORDER BY` clause in a windowed aggregate function. ({issue}`23929`)
+* Improve performance of queries with selective joins on large build sides. ({issue}`22824`)
+* Fix potential configuration leakage between catalogs using JDBC-based
+  connectors. ({issue}`24058`)
+* Reduce chance of out of memory query failure when `retry-policy` is set to
+  `task`. ({issue}`24114`)
+
+# JDBC driver
+
+* Add support for `LocalDateTime` and `Instant` in `getObject` and `setObject`. ({issue}`22906`)
+
+## CLI
+
+* Fix incorrect quoting of output values when the `CSV_UNQUOTED` or the 
+  `CSV_HEADER_UNQUOTED` format is used. ({issue}`24113`)
+
+## BigQuery connector 
+
+* Fix failure when reading views with `timestamp` columns. ({issue}`24004`)
+
+## Cassandra connector
+
+* {{breaking}} Require setting `cassandra.security=PASSWORD` along with
+  `cassandra.username` and `cassandra.password` for password-based
+  authentication. ({issue}`23899`)
+
+## Clickhouse connector
+
+* Fix insert for tables with the Clickhouse `TimestampWithTimeZone` type. ({issue}`23785`)
+
+## Delta Lake connector
+
+* Fix incorrect results when filtering partition columns on checkpoints with
+  `NAME` column mapping mode. ({issue}`24104`)
+
+## Faker connector
+
+* Add the {func}`random_string` catalog function. ({issue}`23990`)
+* Make generated data deterministic for repeated queries. ({issue}`24008`)
+* Allow configuring locale with the `faker.locale` configuration property. ({issue}`24152`)
+
+## Hive connector
+
+* Add support for skipping archiving when committing to a table in the Glue metastore
+  and the `hive.metastore.glue.skip-archive` configuration property is set to
+  `true`. ({issue}`23817`)
+
+## Iceberg connector
+
+* Add support for reading and writing arbitrary table properties with the
+  `extra_properties` table property. ({issue}`17427`, {issue}`24031`)
+* Add the `spec_id`, `partition`, `sort_order_id`, and `readable_metrics`
+  columns to the `$files` metadata table. ({issue}`24102`)
+* Add support for configuring an OAuth2 server URI with the
+  `iceberg.rest-catalog.oauth2.server-uri` configuration property. ({issue}`23086`)
+* Add support to retry connecting to the JBDC catalog with JDBC connection error
```

Comment:
What does "retry ... with JDBC connection error codes" mean? I would simplify the entry and just say:

> Add support for retrying requests to a JDBC catalog.

Reply:
Done but left config property name in place.